api/job_handler.go

```diff
@@ -111,20 +111,110 @@ func (a *Application) PostJobHandler(c echo.Context) error {
 		CreatedBy:    userEmail,
 		CreatedAt:    time.Now().UnixNano(),
 		UpdatedAt:    time.Now().UnixNano(),
+		App:          *app,
 	}
+
 	err = WithSegment("decodeAndValidate", c, func() error {
 		return decodeAndValidate(c, job)
 	})
 	if err != nil {
 		return c.JSON(http.StatusUnprocessableEntity, &Error{Reason: err.Error(), Value: job})
 	}
 
+	skip, err := a.checkFilters(job, c)
+	if err != nil || skip {
+		return err
+	}
+
+	skip, err = a.checkTemplateName(templateName, job, c)
+	if err != nil || skip {
+		return err
+	}
+
+	if job.StartsAt == 0 && job.Localized {
+		localeErr := "Job can not be localized and don't have an start time"
+		return c.JSON(http.StatusUnprocessableEntity, &Error{Reason: localeErr, Value: job})
+	}
+
+	err = WithSegment("create-job", c, func() error {
+		scheduleJob := job.StartsAt
+
+		jobGroup := model.JobGroup{
```

Comment:
This is related to my comment on the migration (you should read it first), but I think we shouldn't use a job group unless we need (i.e. when creating a job for each tz).